setup.cfg

```diff
@@ -35,12 +33,12 @@ packages = find_namespace:
 package_dir =
     =src
 include_package_data = True
-python_requires = >= 3.8
+python_requires = >= 3.10
```

Comment:
why?   we are running it on smaug ATM using system wide python which is just 3.9

```
con@smaug:~$ source /mnt/datasets/con/matrix-archive/tools/.venvs/versations/bin/activate
(versations) con@smaug:~$ python --version
Python 3.9.2
```

Reply:
Because this builds on #33.